recipes/augustus/3.2.2/build.sh

```diff
@@ -19,10 +19,8 @@ mkdir -p $PREFIX/scripts
 mkdir -p $PREFIX/config
 
 sed -i 's/#!\/usr\/bin\/perl/#!\/usr\/bin\/env perl/g' scripts/*
+sed -i 's/ perl -w$/ perl/' scripts/*
 
 mv bin/* $PREFIX/bin/
-mv scripts/* $PREFIX/scripts/
+mv scripts/* $PREFIX/bin/
 mv config/* $PREFIX/config/
-
-#Install perl reqs
-cpanm Cwd File::Basename FileHandle File::Path File::Spec::Functions Getopt::Long IO::File List::Util Term::ANSIColor
```

Comment:
Here you specified many more perl packages than you are actually depending now, is this correct? Just double checking.